packages/crypto-block/source/verifier.ts

```diff
@@ -92,7 +92,7 @@ export class Verifier implements Contracts.Crypto.IBlockVerifier {
 
 			let totalAmount: BigNumber = BigNumber.ZERO;
 			let totalFee: BigNumber = BigNumber.ZERO;
-			let totalPayloadLength = 0;
+			let totalPayloadLength = 4 * block.transactions.length;
```

Comment:
Why is this changed?

Reply:
The payload length was calculated wrong here and for empty blocks this change makes no difference, hence it wasn't spotted earlier.

Reply:
added a comment for clarification

```typescript
result.writeUint32(serialized.length);
result.writeBytes(serialized);
```